### PR TITLE
Moved all coords to vector3 types

### PR DESCRIPTION
**Pull request by rex2630**

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -108,7 +108,7 @@ Citizen.CreateThread(function()
 				local distance = GetDistanceBetweenCoords(playerCoords, v.Pos[i].x, v.Pos[i].y, v.Pos[i].z, true)
```
Comment (ElPumpo) [inline client/main.lua]: could be a vector distance check instead?

Comment (manups4e) [reply client/main.lua]: local distance = #(playerCoords - vector3(v.Pos[i].x, v.Pos[i].y, v.Pos[i].z))

Comment (rex2630) [reply client/main.lua]: thanks

Comment (manups4e) [reply client/main.lua]: my pleasure

File: client/main.lua [hunk 2]
```diff
@@ -105,7 +105,7 @@ Citizen.CreateThread(function()
 
 		for k,v in pairs(Config.Zones) do
 			for i = 1, #v.Pos, 1 do
-				local distance = GetDistanceBetweenCoords(playerCoords, v.Pos[i].x, v.Pos[i].y, v.Pos[i].z, true)
+				local distance = #(playerCoords - vector3(v.Pos[i].x, v.Pos[i].y, v.Pos[i].z))
```
Comment (ElPumpo) [inline client/main.lua]: wtf is this? `v.Pos[i]` is already a `vector3` type no need to create a new one?

`local distance = #(playerCoords - v.Pos[i])`

Comment (rex2630) [reply client/main.lua]: I dont know why i didnt checked 🤦‍♂

Comment (ElPumpo) [reply client/main.lua]: No problem funny how manups who apparently is so much better of a dev fail things like this and idk why he even is active here but constantly trash talking esx it makes no sense

File: config.lua [hunk 3]
```diff
@@ -1,49 +1,46 @@
...
+Config = {}
 Config.DrawDistance = 100
-Config.Size         = {x = 1.5, y = 1.5, z = 1.5}
-Config.Color        = {r = 0, g = 128, b = 255}
-Config.Type         = 1
-Config.Locale       = 'en'
+Config.Size = {x = 1.5, y = 1.5, z = 1.5}
+Config.Color = {r = 0, g = 128, b = 255}
+Config.Type = 1
+Config.Locale = 'fr'
```
Comment (Dislaik) [inline config.lua:6]:
```suggestion
Config.Locale = 'en'
```

**Pull request merged**